### DISCUSSION TITLE
HUGO用Qexo创建文章必须在posts路径下吗

### DISCUSSION THREAD

**Discussion by rvnox** [Qexo/Qexo#517] [Q&A] [answered]
用qexo创建的文章必须是创建在content/posts/下吗，可以创建 content/a/1.md 或者 content/b/1.md 吗

**Comment by am-abudu** [comment]
子目录直接打在filename里面
例如 dir/1

**Reply by rvnox** [reply]
好像还是不行呀，这样是创建了 content/posts/dir/1.md ,我想创建 content/dir/1.md

**Reply by am-abudu** [reply]
那确实是不行的。请告诉我为什么你会有这个需求

**Reply by rvnox** [reply]
![image](https://github.com/Qexo/Qexo/assets/67718036/215078b2-c3a8-4e62-b548-2f36f29ff7d0)

 这是我的content目录，面包屑显示直接如下所示：
 
 
![image](https://github.com/Qexo/Qexo/assets/67718036/4142dcda-5618-4a14-8ad3-5e4d05d90798)

**Comment by am-abudu** [comment, marked as answer]
你可以自行修改配置文件hugo.py来完成这个目的。
